common/script/ops/openMysteryItem.js

```diff
@@ -28,17 +27,13 @@ module.exports = function openMysteryItem (user, req = {}, analytics) {
     });
   }
 
-  if (typeof window !== 'undefined') {
-    if (!user._tmp) user._tmp = {};
-    user._tmp.drop = item;
-  }
-
   if (req.v2 === true) {
     return user.items.gear.owned;
   } else {
     return [
       user.items.gear.owned,
       i18n.t('mysteryItemOpened', req.language),
+      item,
```

Comment:
That way, if people want a quick an easy way to overwrite the `gear.owned` user prop, they can.

I'm fine with just returning the item though. Returning an object with properties is more forwards compatible.